common.lic

```diff
@@ -369,13 +369,21 @@ module DRC
         .sub('.', ' ') # convert 'foo.bar' => 'foo bar' so more easily split into adjective and noun
         .squeeze(' ') # condense repeated runs of whitespace with a single space
         .strip # remove leading/trailing whitespace
+      has_my_prefix = text.start_with?('my ')
+      text = text.delete_prefix('my ')
       parts = text.split
       if text.nil? || text.empty?
         nil
       elsif parts.size > 1
-        DRC::Item.new(adjective: parts.first, name: parts.last)
+        DRC::Item.new(
+          adjective: "#{has_my_prefix ? 'my ' : ''}#{parts.first}",
```

Comment:
Baking the 'my' into an item adjective feels like it will have repercussions elsewhere. The item should be the item, and the method using the item should probably decide to use 'my' or not, IMO.

Reply:
Gotcha, thanks for the feedback